tests/bears/.coafile

```diff
@@ -0,0 +1,3 @@
+[cli]
+files = BearTest.py
+bears = SpaceConsistencyBear
```

Comment:
hm yeah what is this file actually for? :D

Reply:
I think it got created when I ran coala on BearTest.py, I completely forgot about that. I'll delete it then push again.